### PR TITLE
fix the priority of stdin and in

### PR DESCRIPTION
**Pull request by feich-ms**
fix issue #790

### CODE REVIEW

File: packages/lu/src/utils/filehelper.ts [hunk 1]
```diff
@@ -17,15 +17,15 @@ const globby = require('globby')
 
 export async function getLuObjects(stdin: string, input: string | undefined, recurse = false, extType: string | undefined) {
   let luObjects: any = []
-  if (stdin) {
-    luObjects.push(new luObject(stdin, new LUOptions('stdin')))
-  } else {
+  if (input) {
```
Comment (munozemilio) [inline packages/lu/src/utils/filehelper.ts:20]: If I remember correctly stdin takes precedence over input in other commands. We should have the same behavior across the commands...@vishwacsena thoughts?

Comment (feich-ms) [reply packages/lu/src/utils/filehelper.ts:20]: How about making input precedent over stdin in all cli commands to keep the consistency?  It seems blocking issue for customers if we don't take this change. Do you have better solutions? @munozemilio @vishwacsena

Comment (vishwacsena) [reply packages/lu/src/utils/filehelper.ts:20]: Looking at the issue we are attempting to fix, I agree we should have `--in` supersede anything passed in via `stdin`. @feich-ms, can we also via this PR update chatdown:convert as well as qnamaker:convert to prefer `--in` over `stdin`? Also do this for `lg` commands? I cannot think of other places where we support reading from stdin but would be great if you can scan the BF-CLI code and update all places.

Comment (feich-ms) [reply packages/lu/src/utils/filehelper.ts:20]: sure. will make changes for all occurrences.

Comment (feich-ms) [reply packages/lu/src/utils/filehelper.ts:20]: All occurrences are resolved

**Comment by vishwacsena** [comment]
@feich-ms this seems to be missing updates to chatdown..?

**Comment by feich-ms** [comment]
@vishwacsena, I scanned the chatdown code, found that the --in already has higher priority than stdin there and lg tool does not accept stdin.

**Comment by feich-ms** [comment]
@vishwacsena, does this look good to go for you?

**Pull request merged**